noir-projects/noir-contracts/contracts/app/orderbook_contract/src/main.nr

```diff
@@ -166,23 +165,16 @@ pub contract Orderbook {
         order_id: Field,
         taker_partial_note: PartialUintNote,
         bid_token: AztecAddress,
-        ask_token: AztecAddress,
         bid_amount: u128,
-        ask_amount: u128,
     ) {
-        // The `order_id` is a serialized form of the maker's partial note.
-        let maker_partial_note = PartialUintNote::from_field(order_id);
-
-        // Finalize transfer of ask_amount of ask_token to maker
-        Token::at(ask_token).finalize_transfer_to_private(ask_amount, maker_partial_note).call(
-            &mut context,
-        );
-
         // Finalize transfer of bid_amount of bid_token to taker
         Token::at(bid_token).finalize_transfer_to_private(bid_amount, taker_partial_note).call(
             &mut context,
         );
 
+        // TODO: This would make sense to emit in `fulfill_order` as then we would not have to pass it to this function
+        // and we would do less args hashing. But as far as I know emitting of unencrypted events in private is not
+        // supported.
```

Comment:
There is no notion of 'unencrypted events': events are either emitted in private (typically encrypted) or in public (unencrypted). They can then be found from either source.

But private and public behave differently because private logs get siloing and have no known emitter, while public logs have no siloing and a known emitter. And no, we can't emit public logs from private.